apps/cb-ceiling/.eslintrc.json

```diff
@@ -45,9 +45,10 @@
     "prefer-destructuring": "warn",
     "no-param-reassign": "warn",
     "camelcase": "warn",
-    "eqeqeq": "warn",
     "no-redeclare": "warn",
     "no-shadow": "warn",
+    "no-useless-escape": "warn",
+    "import/no-unresolved": "warn",
```

Comment:
It's great to see that we're now enforcing `eqeqeq`. However, our end goal is to promote all these warnings to full errors so it'd be ideal if we avoided adding new eslint exceptions. Is there a reason we have to have these?

Here's the `no-useless-escape` docs: https://eslint.org/docs/latest/rules/no-useless-escape

Here are the `import/no-unresolved` docs: https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-unresolved.md

Reply:
I got them from the ceil-vis eslint file, thinking I'd deal with them all at once. We probably don't need them here, but it better matches the other apps' linter files now. I'll remove them if you want.

Reply:
Gotcha.

Some background on my thinking here - I scoped each app's eslint exceptions to just the errors present in the code for that app. This means the list of ESLint exceptions in the eslint config file will differ between apps. By doing that, I was hoping that would limit the number of new ESLint errors we needed to spend time resolving later. 

Given you're planning on doing the work to resolve the linter errors I'm okay with however you want to handle this, Molly. 🙂

Reply:
Sounds good, I think most of the MATS apps were coincidentally identical, then, which makes sense. cb-ceiling is by far the most different app. I'll remove it and push to this branch again.

Reply:
Ah, yeah. I think there were usually 2-4 rules that differed between the apps at most. As you pointed out, the code is very similar between them all! 😁

Reply:
Done.